frameworks/JavaScript/koa/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
you wont need gitignores anymore since the test will be built and run in a docker container